blockchain/standalone/subsidy.go

```diff
@@ -338,7 +338,7 @@ func (c *SubsidyCache) CalcTreasurySubsidy(height int64, voters uint16) int64 {
 
 	// Ignore any potential subsidy reductions due to the number of votes prior
 	// to the point voting begins.
```

Comment:
to the point voting begins or if treasury is enabled.

Reply:
Fixed.